### PR TITLE
Ensure Terms link to their appropriate Taxonomy

### LINKED ISSUE #1296: The Terms response should include a link property to the Taxonomy
While reviewing #1284 I noticed the terms response doesn't include a _link to the taxonomy.

### PR DESCRIPTION
**Pull request by danielbachhuber**
Fixes #1296

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@WP-API/amigos #reviewmerge

**Comment by danielbachhuber** [comment]
```
local ➜  ~  http GET wordpress-develop.dev/wp-json/wp/v2/categories/1
HTTP/1.1 200 OK

{
    "_links": {
        "about": [
            {
                "href": "http://wordpress-develop.dev/wp-json/wp/v2/taxonomies/category"
            }
        ],
        "collection": [
            {
                "href": "http://wordpress-develop.dev/wp-json/wp/v2/categories"
            }
        ],
        "self": [
            {
                "href": "http://wordpress-develop.dev/wp-json/wp/v2/categories/1"
            }
        ]
    },
    "count": 111,
    "description": "",
    "id": 1,
    "link": "http://wordpress-develop.dev/blog/category/uncategorized/",
    "name": "Uncategorized",
    "parent": 0,
    "slug": "uncategorized",
    "taxonomy": "category"
}
```

File: lib/endpoints/class-wp-rest-terms-controller.php [hunk 1]
```diff
@@ -484,6 +484,9 @@ protected function prepare_links( $term ) {
 			'collection' => array(
 				'href'       => rest_url( $base ),
 			),
+			'about'      => array(
+				'href'       => rest_url( sprintf( 'wp/v2/taxonomies/%s', $this->taxonomy ) ),
+				),
```
Comment (rmccue) [inline lib/endpoints/class-wp-rest-terms-controller.php]: Closing paren is off ;)

**Comment by rmccue** [comment]
lgtm, needs standards tweak

**Pull request merged**